zerver/worker/queue_processors.py

```diff
@@ -59,6 +59,7 @@
 import re
 import importlib
 
+logger = logging.getLogger('zerver.worker.queue_processors')
```

Comment:
Let's use `__name__` here -- that way it's totally clear where the name comes from, and also this code becomes copy-paste safe. :-) (Also does the right thing, with no extra mucking about required, if we rename the file.)